tools/perf_k8svcs.yaml

```diff
@@ -31,6 +31,21 @@ spec:
         imagePullPolicy: Always # needed despite what is documented to really get latest
         ports:
         - containerPort: 8080
+        - containerPort: 12000
```

Comment:
I'm missing something, who listens to 12000, how is this working ? I don't see any change using it so this would just fail ?